core/src/test/java/org/apache/accumulo/core/util/PairTest.java

```diff
@@ -26,13 +30,42 @@
 public class PairTest {
 
   /**
+   * Test method for {@link org.apache.accumulo.core.util.Pair#hashCode()}.
+   */
+  @Test
+  public void testHashMethod() {
+    Pair<Integer,String> pair1 = new Pair<>(25, "twenty-five");
+    Pair<Integer,String> pair2 = new Pair<>(25, "twenty-five");
+    Pair<Integer,String> pair3 = new Pair<>(null, null);
```

Comment:
Could add check the case where "first" is equal, but "second" is different and vice-versa.